### ISSUE TITLE
 Unable to load default MotionControllers model

### ISSUE THREAD

**Issue by jerrygg** [Microsoft/MixedRealityToolkit-Unity#3119]
**Does this affect the legacy HoloToolkit (master) or the Mixed Reality Toolkit (mrtk_release)?**
HoloToolkit 

**Describe the bug**
 Unable to load default handle model，The console reports the error as follows.

Unable to find element of type PointingPose under controller RightController; not attaching.
```
UnityEngine.Debug:LogError(Object)
HoloToolkit.Unity.InputModule.ControllerFinder:AddControllerTransform(MotionControllerInfo) (at Assets/HoloToolkit/Input/Scripts/Utilities/ControllerFinder.cs:134)
```

**To Reproduce**
1. open BoundingBoxGizmoExample.scenes
2. click play
3. Press the MotionControllers trigger

**Expected behavior**
load default MotionControllers model

**Actual behavior**
 Unable to load default MotionControllers model

**Screenshots**
![qq 20181112163728](https://user-images.githubusercontent.com/7133614/48335638-4eba0a00-e699-11e8-948e-030fdcc24869.png)

**Unity Editor Version**
Unity 2017.4.13f

**Mixed Reality Toolkit Release Version**
Master

**Comment by marklinmao** [comment]
Looks like the same problem as this one?
https://github.com/Microsoft/MixedRealityToolkit-Unity/issues/2861

**Comment by jerrygg** [comment]
@marklinmao Yes, I met in Unity 2017.4.13.

**Comment by keveleigh** [comment]
This should be fixed in the 2017.4.3 release. Please reopen if not.
